### DISCUSSION TITLE
Where to go next

### DISCUSSION THREAD

**Discussion by artesim-zz** [Ortho4XP/Ortho4XP#26] [General]
Hi,

We've each had a lot of ideas since we created this community repo, and I
propose we start to dump them in the [Ideas section](https://github.com/Ortho4XP/Ortho4XP/discussions/categories/ideas)

This would give us a better overall picture of what we want to do.

Also, the next step would be to create Draft PRs[1], so this first step would
also help us regrouping (or splitting) ideas into better PRs.

Another benefit is that this would help newcomers to see where they could help.

Finally, I guess we all have limited spare time, so this would also help any
potential new contributor to choose a subject on which to focus on.
(speaking of which, I will probably not be much available this week-end)

[1] IMO Draft PRs are better than issues for this use case, I think issues should
    be reserved for... well, for real issues ;-)

As you can see, I took a head start :)

**Comment by girotobial** [comment]
I don't necessarily agree about reserving issues for only _real issues_. Personally I see github issues as a "TODO list" of which bugs/problems are only part.

**Reply by artesim-zz** [reply]
My main point was more that we should not use issues for things that could be Draft PRs

But yes, you're right, issues could also be used as todo.

Although, now that I discovered it, I think my preference would go to a card in something like this https://github.com/orgs/Ortho4XP/projects/1 (I was only trying it by the way, we could change the name of the kanban, we could only keep one and remove [Bug Triage](https://github.com/orgs/Ortho4XP/projects/2), or not use it at all...)

So many choices to do the same thing :)
